.coveralls.yml

```diff
@@ -0,0 +1 @@
+repo_token: 0CzkQAXABFpjYwwk9l0sCx1y0hwtfGxmC
```

Comment:
https://github.com/lemurheavy/coveralls-public/issues/208

You might need to generate a new token